modules/coreos-layering-configuring.adoc

```diff
@@ -24,20 +24,21 @@ When you configure a custom layered image, {product-title} no longer automatical
 You should use the same base {op-system} image that is installed on the rest of your cluster. Use the `oc adm release info --image-for rhel-coreos-8` command to obtain the base image being used in your cluster.
 ====
 +
-For example, the following Containerfile creates a custom layered image from an {product-title} 4.12 image and a Hotfix package:
+For example, the following Containerfile creates a custom layered image from an {product-title} 4.12 image and overrides the kernel package with one from CentOS 8 Stream:
 +
-.Example Containerfile for a custom layer image
+.Example Containerfile for a custom layer image with a kernel override
 [source,yaml]
 ----
 # Using a 4.12.0 image
 FROM quay.io/openshift-release/ocp-release@sha256... <1>
 #Install hotfix rpm
-RUN rpm-ostree override replace https://example.com/hotfixes/haproxy-1.0.16-5.el8.src.rpm && \ <2>
+RUN rpm-ostree cliwrap install-to-root / \
+    rpm-ostree override replace http://mirror.centos.org/centos/8-stream/BaseOS/x86_64/os/Packages/kernel-{,core-,modules-,modules-extra-}.4.18.0-483.el8.x86_64.rpm && \ <2>
```

Comment:
Another wrinkle here is that because 4.13+ use RHEL9 content, we should be documenting using a RHEL9 kernel there, not RHEL8.

In https://github.com/openshift/rhcos-image-layering-examples we have `override-kernel-4.12` and `override-kernel` for this reason.

Most of the other examples will generally work across 4.12/4.13, although not always.  But the kernel one is more special.